src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -0,0 +1,158 @@
+// Metadata version: v4.0.30319
+.assembly extern System.Runtime
+{
+  .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A )                         // .?_....:
```

Comment:
Maybe the comments in this file could be kept to 1 space after the actual code (to make it more readable)? On GitHub you have to scroll to see the end of some comments.